lib/dao.js

```diff
@@ -1537,6 +1537,9 @@ DataAccessObject._normalize = function(filter, options) {
 };
 
 function DateType(arg) {
+  if (arg == null || arg == "null") {
```

Comment:
I wonder how a nested date type filter with a null will work with this code: 
`http://localhost:3000/api/MyThing?filter={"where":{"some_date_property":{"neq":"null"}}}`?